################################################################################
SHORT DESCRIPTION: 
################################################################################
Change version name to custom "openxt".

################################################################################
LONG DESCRIPTION: 
################################################################################
Output that the SeaBIOS version booted is the one including OpenXT
modifications.

################################################################################
CHANGELOG 
################################################################################
Ported to SeaBIOS 1.7.5: Eric Chanudet, chanudete@ainfosec.com, 01/03/2015

################################################################################
REMOVAL 
################################################################################
This patch is only cosmetic, or for testing purpose.

################################################################################
UPSTREAM PLAN 
################################################################################
This is OpenXT specific. There is no plan to upstream this patch.

################################################################################
INTERNAL DEPENDENCIES 
################################################################################
None.

################################################################################
PATCHES
################################################################################
--- a/scripts/buildversion.sh
+++ b/scripts/buildversion.sh
@@ -11,7 +11,7 @@ elif [ -f .version ]; then
 else
     VERSION="?"
 fi
-VERSION="${VERSION}-`date +"%Y%m%d_%H%M%S"`-`hostname`"
+VERSION="${VERSION}-openxt"
 echo "Version: ${VERSION}"
 
 # Build header file
